sklearn/linear_model/_glm/tests/test_glm.py

```diff
@@ -269,14 +268,6 @@ def test_glm_regression_hstacked_X(solver, fit_intercept, glm_dataset):
         max_iter=1000,
     )
 
-    if solver == "lbfgs" and not fit_intercept:
-        # Sometimes (depending on global_random_seed) lbfgs fails with:
-        # Line search cannot locate an adequate point after MAXLS function and gradient
-        # evaluations. Previous x, f and g restored.
-        # Possible causes: 1 error in function or gradient evaluation;
-        #                  2 rounding error dominate computation.
-        pytest.xfail()
-
```

Comment:
This would no longer fail, so I removed the xfail. Not sure if it's related to the change of this PR or if it is a consequence of the increased MAXLS setting in [79ec862](https://github.com/scikit-learn/scikit-learn/pull/23619/commits/79ec862e0bc0d8c6c5bde8b0253c1ebde4355ae0).

Reply:
I think it's because of this PR.